src/ua_parser/_matchers.pyi

```diff
@@ -0,0 +1,3 @@
+from .core import Matchers
+
+MATCHERS: Matchers
```

Comment:
Should probably be typed more concretely, so that a parser has access to better details (e.g. the bulk/batch parsers need to get the regex string out of the matcher to compose them into sets)